api/services/idam.ts

```diff
@@ -41,6 +43,7 @@ export async function getUser(email = null) {
 
 export async function postOauthToken(code, host) {
     const redirectUri = `${idamProtocol}://${host}/${oauthCallbackUrl}`
+    logger.info(`idam details ${idamClient}:${idamSecret}`)
```

Comment:
Is it a good idea to log idam secrets? Especially at info level.

Reply:
I see you removed it in later commit

Reply:
Its fine until it goes into prod from a sec point of view, but these details were only put into help debug what was going on (very usefully too), so I removed them in subsequent commits.